Source/Editor/DualityEditor/Controls/AutoExpandTreeView.cs

```diff
@@ -0,0 +1,68 @@
+using Aga.Controls.Tree;
+using System;
+using System.Drawing;
+using System.Windows.Forms;
+
+namespace Duality.Editor.Controls
+{
+	public class AutoExpandTreeView : TreeViewAdv
+	{
+		public int DragOverExpandDelay
+		{
+			get;
+			set;
+		} = 1000;
+
+		private TreeNodeAdv autoExpandNode = null;
+
+		private Timer timer = new Timer();
+
+		private void StartTimer()
+		{
+			if (DragOverExpandDelay > 0)
```

Comment:
Something else: All member access (fields, properties, methods) needs to be preceded with a `this.` - this is a code convention in the Duality repository.